### PR TITLE
Edit User Guide to add Member to Event & Delete Member from Event features and command summary

### LINKED ISSUE #61: Edit User Guide to add Member to Event & Delete Member from Event features and command summary

### PR DESCRIPTION
**Pull request by ph-nathan**
closes #61 
* Edited user guide to introduce these following changes:
  * Added title header to user guide file 
  * Added add a member  to event feature and command summary
  * Added delete member from event feature and command summary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -137,6 +137,35 @@ Examples:
 * `delete /event 1 ` deletes the 1st event in the event list.
 * `delete /event 10 ` deletes the 10th event in the event list.
 
+### Add Member to an Event: `add /member /event`
+
+Adds a member to an event. 
+
+Format: `add /member MEMBER_INDEX /event EVENT_INDEX`
+
+* Add the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX`.
```
Comment (andrechuakj) [inline docs/UserGuide.md]: I think the correct grammar should be "adds"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -137,6 +137,35 @@ Examples:
...
+* The member/event index refers to the index number shown in the displayed member/event list.
+* The member/event index **must be a positive integer** that is within the range of the length of the member/event list.
```
Comment (andrechuakj) [inline docs/UserGuide.md]: Should we follow the `MEMBER_INDEX` and `EVENT_INDEX` format instead of "member/event index" to ensure consistency

**Review by andrechuakj — COMMENTED** [review]
just some nits

**Comment by andrechuakj** [comment]
LGTM!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -137,6 +137,35 @@ Examples:
...
+### Delete Member from an Event: `delete /member /event`
+
+Deletes a member from an event.
+
+Format: ` delete /member MEMBER_INDEX /event EVENT_INDEX`
+
+* Delete the member at the specified `MEMBER_INDEX` from the event at the specified `EVENT_INDEX`.
+* The member at `MEMBER_INDEX` must be a part of the event at `EVENT_INDEX`
+* The member/event index refers to the index number shown in the displayed member/event list.
+* The member/event index **must be a positive integer** that is within the range of the length of the member/event list.
```
Comment (andrechuakj) [inline docs/UserGuide.md]: same nits as above

**Review by andrechuakj — COMMENTED** [review]
similar nits as previous

File: docs/UserGuide.md (hunk 3 above)
Comment (ph-nathan) [reply docs/UserGuide.md]: oh oof sorry bout that :') Fixed!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: User Guide
 ---
-
+## CACommander Ultra Promax Xtra 9000PLUS - User Guide
```
Comment (SelwynAng) [inline docs/UserGuide.md]: I think there is a spelling error here, should be "CCACommander" not "CACommander"

**Comment by SelwynAng** [comment]
LGTM!

**Pull request merged**